### ISSUE TITLE
Table of contents not showing up

### ISSUE THREAD

**Issue by PythonCoderAS** [squidfunk/mkdocs-material#3110]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

The table of contents does not show up at all.

### Expected behaviour

The table of contents to show up.

### Actual behaviour

There is no table of contents in the sidebar.

### Steps to reproduce

1. Use material theme
2. Generate a simple site with 2 headings
3. Build

### Package versions

- Python: `Python 3.9.7`
- MkDocs: `mkdocs, version 1.2.2 from /Users/<User>/.local/share/virtualenvs/docs-OQy_yQY0/lib/python3.9/site-packages/mkdocs (Python 3.9)`
- Material: `Version: 7.3.2`

### Configuration

```yaml
site_name: Health Screening Bot
theme:
  name: material
markdown_extensions:
  - admonition
  - toc:
      permalink: false
```

### System information

- Operating system: MacOS
- Browser: Google Chrome 94

**Comment by PythonCoderAS** [comment]
> Possibly related:
> 
> * [TOC does not list all h1 & children elements #818 (comment)](https://github.com/squidfunk/mkdocs-material/issues/818#issuecomment-629646709)

This might be it since I use a h1 to title the page, and then all of my headers are H1's.

**Comment by facelessuser** [comment]
Yes, you need to only have one H1 on a page. This is a known behavior with MkDocs.
